manager/catalog.go

```diff
@@ -320,6 +320,12 @@ func readFile(relativePath string, fileName string) (*[]byte, error) {
 		log.Errorf("Error forming path to file %s, error: %v", relativePath+"/"+fileName, err)
 	}
 
+	_, err = os.Stat(filename)
+	if os.IsNotExist(err) {
+		log.Debugf("File %s does not exist", relativePath+"/"+fileName)
```

Comment:
Can you use `path.Join` for this and save it to a variable so it doesn't have to be done multiple times?